README.md

```diff
@@ -443,7 +443,9 @@ Even in an age where we trust our portable devices with the most private data, a
 
 If your app needs to store sensitive data, such as a username and password, an authentication token or some personal user details, you need to keep these in a location where they cannot be accessed from outside the app. Never use `NSUserDefaults`, other plist files on disk or Core Data for this, as they are not encrypted! In most such cases, the iOS Keychain is your friend. If you're uncomfortable working with the C APIs directly, you can use a wrapper library such as [SSKeychain][sskeychain] or [UICKeyChainStore][uickeychainstore].
 
-When storing files and passwords, be sure to set the correct protection level, and choose it conservatively. If you need access while the device is locked (e.g. for background tasks), use the "accessible after first unlock" variety. In other cases, you should probably require that the device is unlocked to access the data. Only keep sensitive data around while you need it.
+When storing files and passwords, be sure to set the correct protection level, and choose it conservatively. If you need access while the device is locked (e.g. for background tasks), use the "accessible after first unlock" variety. In other cases, you should probably require that the device is unlocked to access the data.
```

Comment:
Good point, that might sound confusing. I suppose what was originally meant by the writer was that no matter which data you store – whether it's in the keychain or not – you need to think about the security level that you need, and do so rather conservatively. I would simply rephrase "files and passwords" to "data" and avoid getting into specifics.